filebeat/crawler/prospector.go

```diff
@@ -242,3 +299,20 @@ func getConfigDuration(config string, duration time.Duration, name string) (time
 
 	return duration, nil
 }
+
+// isIgnoreOlder checks if the given state reached ignore_older
+func (p *Prospector) isIgnoreOlder(state input.FileState) bool {
+
+	// ignore_older is disable
+	if p.ProspectorConfig.IgnoreOlderDuration == 0 {
```

Comment:
can rename this to IgnoreOlder at some point? (btw. ucfg supports unpacking time.Duration out of the box).